### DISCUSSION TITLE
I can't install lanraragi 0.8.81 and 0.8.8 for windows.

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#735] [Q&A]
I can't install lanraragi 0.8.81 and 0.8.8 for windows. Always prompt lanraragi setup wizard was interrupted. Even if I have uninstalled the old version, even if I have switched the system language to English, it is still useless.

**Comment by Difegue** [comment]
Please follow the troubleshooting instructions [here](https://sugoi.gitbook.io/lanraragi/v/dev/installing-lanraragi/windows#installer-failures) and report back with a MSI log if the problem isn't fixed. 🤔
